scripts/utils/update_package.py

```diff
@@ -37,7 +37,10 @@ def get_latest_version(org, project, version):
         print(f"GitHub API response not ok: {response.status_code}")
         return None
     latest_version = response.json()["tag_name"]
-    return latest_version
+    if latest_version.startswith('v'):
+    	return latest_version.replace('v','',1)
```

Comment:
A bit more elegant :wink: 
```suggestion
    	return latest_version[1:]
```